module/functions/azure/Invoke-ArmTemplateDeployment.ps1

```diff
@@ -58,15 +61,17 @@ function Invoke-ArmTemplateDeployment
     [CmdletBinding()]
     param
     (
-        [Parameter(Mandatory=$true)]
-        [string] $ResourceGroupName,
-
         [Parameter(Mandatory=$true)]
         [string] $Location,
 
         [Parameter(Mandatory=$true)]
         [string] $ArmTemplatePath,
 
+        [ValidateSet("ResourceGroup","Subscription","Tenant")]
```

Comment:
Any reason not to include `ManagementGroup`?

Reply:
Nope, an oversight - thanks!